doc/user-guide/Getting-started.md

```diff
@@ -16,34 +16,30 @@ MongooseIM is supported on Mac OS X 10.6.8 and later. Before you can compile and
 
 An easy way to install some of the packages is by using a package manager, such as [Homebrew](http://brew.sh) – the Homebrew commands are provided here.
 
-
 ##### Build and install from source code
 
 To build and install MongooseIM from source code, do the following:
 
-1. Clone the Git repository:  `git clone https://github.com/esl/MongooseIM.git`
+1. Clone the Git repository: `git clone https://github.com/esl/MongooseIM.git`
 2. Go to your MongooseIM directory.
 3. Run the following command: `make rel`.
 
 For more advanced release generation and installation please see [Release/Installation configuration](release_config.md)
 
-
 #### Install from DMG
 
 Go to the [downloads](https://www.erlang-solutions.com/resources/download.html) page of Erlang Solution website, and download the version of MongooseIM you want.
 Once the DMG is downloaded, double click on it, it will open the content.
 Double click on the .pkg file, it will fire an installation wizard.
 Just follow the instructions.
 
-
 #### Running MongooseIM
 
-* From your MongooseIM directory, go to the release directory:  `cd _build/prod/rel/mongooseim`
-* To start the MongooseIM server, run the following command:  `bin/mongooseim start`
-* To verify that MongooseIM is running, enter the following:  `bin/mongooseim ping`<br \>If the response is `pong`, then MongooseIM is running.
-* To connect to the MongooseIM console after starting the server:  `bin/mongooseim debug`
-* Alternatively, you can also run the server in interactive mode:  `bin/mongooseim live`
-
+* From your MongooseIM directory, go to the release directory: `cd _build/prod/rel/mongooseim`
+* To start the MongooseIM server, run the following command: `bin/mongooseim start`
+* To verify that MongooseIM is running, enter the following: `bin/mongooseim ping`<br>If the response is `pong`, then MongooseIM is running.
+* To connect to the MongooseIM console after starting the server: `bin/mongooseim debug`
```

Comment:
To connect to the MongooseIM console after starting the server, use: `bin/mongooseim debug`